docs/en/observability/observability-ai-assistant.asciidoc

```diff
@@ -231,6 +231,65 @@ image::images/obs-ai-logs.gif[Observability AI assistant example, 75%]
 
 You can continue a conversation from a contextual prompt by clicking *Start chat* to open the AI Assistant chat.
 
+[discrete]
+[[obs-ai-connector]]
+=== AI Assistant connector
+
+//TODO: After https://github.com/elastic/kibana/pull/183792 is merged, make "configure the Observability AI Assistant connector" an active link to the published docs.
+
+IMPORTANT: To use the Observability AI Assistant connector,
+you must have the `api:observabilityAIAssistant` and `app:observabilityAIAssistant` privileges.
+
+You can use the Observability AI Assistant connector to add AI-generated insights and custom actions to your alerting workflows.
+To do this:
+
+. <<create-alerts-rules,Create (or edit) an alerting rule>> and specify the conditions that must be met for the alert to fire.
+. Under **Actions**, select the **Observability AI Assistant** connector type.
+. In the **Connector** list, select the AI connector you created when you set up the assistant.
+. In the **Message** field, specify the message to send to the assistant:
++
+[role="screenshot"]
+image::images/obs-ai-assistant-action-high-cpu.png[Add an Observability AI assistant action while creating a rule in the Observability UI]
+
+You can ask the assistant to generate a report of the alert that fired,
+recall any information or potential resolutions of past occurrences stored in the knowledge base,
+and also include other active alerts that may be related.
+As a last step, you can ask the assistant to trigger an action,
+such as sending the report (or any other message) to a Slack webhook.
+
+NOTE: Currently you can only send messages to a Slack webhook.
+Support for additional actions will be added in the future.
```

Comment:
Other connectors are supported in the code ([list here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/observability_ai_assistant_app/server/rule_connector/index.ts#L44-L51)) but were either shallowly tested or not tested at all. Since this is tech preview we could still mention them here

Reply:
I'll add them and also remove the word "support" because that might imply that all these actions have been tested. :-)